client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -97,6 +97,27 @@ export const doDismissJITM = ( action ) =>
 		action
 	);
 
+/**
+ * Dismisses a jitm on the jetpack site, it returns nothing useful and will return no useful error, so we'll
```

Comment:
Could we explain the `doDismissJITMDirect` in more detail to identify the difference from `doDismissJITM`?

Reply:
Good point. Added comment in 75d9257